### PR TITLE
Redesign page heading

### PR DESCRIPTION
**Pull request by zdroid**
There are some apparent problems with the front page's heading, the most visible thing on it. It gives the feel of some unmaintained, poorly-designed website. The problems are:

1. Lack of Julia's description
2. Poorly spaced buttons
3. Bad button styles (e.g. green button's border)

I recalled the phrase "We are greedy: we want more" and decided to redesign the page heading completely.

Before:
Screenshot from 2020-05-14 18-43-40
After:
Screenshot from 2020-05-14 19-03-59

Some of @cormullion's SVG banners also style very nicely in this layout. Examples:
Screenshot from 2020-05-14 19-16-14
Screenshot from 2020-05-14 19-16-01
Screenshot from 2020-05-14 19-15-39

### CODE REVIEW

**Comment by abhishalya** [comment]
Some space between the two buttons? Also how about we add an icon to the download button.

**Comment by zdroid** [comment]
Like this?
Screenshot from 2020-05-14 19-56-17

To implement the icon idea, it may be handy to remove the v1.4 from the button text.

**Comment by ViralBShah** [comment]
I like this! Should we go with the triangles?

**Comment by zdroid** [comment]
@ViralBShah The first of @cormullion's SVG banners that I posted? That one caught my eye too, it's quite attractive.

**Comment by cormullion** [comment]
How does the asymmetry work on mobile or when resized?

**Comment by ViralBShah** [comment]
Do remove the dot at the end of the second line.

**Comment by zdroid** [comment]
@ViralBShah Are you sure about removing the dot? It feels somewhat more serious with a period, at least in my judgement. Go, Kotlin and Python also have a description as a full sentence, with a period.

Also, split buttons with a space in between or not?

@cormullion Pretty good, and definitely better than it is currently.
Screenshot from 2020-05-14 20-34-39

**Comment by ViralBShah** [comment]
I feel it would be better without the dot. I like the space between the two buttons.

**Comment by cormullion** [comment]
By the way - your "some unmaintained, poorly-designed website" comment is a bit unfair in my view; volunteers have worked quite hard on this site. It's obviously easier to tweak the details such as button spacing at a later date. My $0.02.

**Comment by zdroid** [comment]
@cormullion It wasn't meant to belittle the work done on the website. The site obviously has a lot good things, but the first impression is the most important for newcoming users. No matter how much good content the site has, if the first impression is bad, it won't catch their attention. :cry:  That's why I proposed this redesign in the first place.

**Comment by tlienart** [comment]
this is great :+1: we should get more people to chip in, could you post a screenshot in mobile mode?

**Comment by zdroid** [comment]
@tlienart it's above: https://github.com/JuliaLang/www.julialang.org/pull/810#issuecomment-628815221

**Comment by StefanKarpinski** [comment]
I really love this banner:

Screenshot from 2020-05-14 19-16-14

And the new design looks really nice!

**Comment by zdroid** [comment]
Due to the increased interest for that one splash image, I put it on and tweaked the other elements to be styled better (text-shadow, box-shadow). Here it is:

Screenshot from 2020-05-15 00-52-17

The only problem on this page heading is this GitHub star button. It is impossible to style it properly without editing the button-loading script. We can:
1. either keep it like this, even though doesn't have proper box-shadow and vertical alignment (it's a bit skewed towards the top)
2. or replace it with our own custom button.

**Comment by ViralBShah** [comment]
Can we try our own version of the star button?

**Comment by zdroid** [comment]
@ViralBShah I think that will go a bit harder. Another option which I forgot is to move the star button somwhere else, like the Open source section right below.

**Comment by ViralBShah** [comment]
BTW, I am ok with the button styling as is.

**Comment by zdroid** [comment]
I pushed the new version now, in case anyone wants to try it out. It's in the `New background image` commit, so if it gets rejected I can easily go back to the previous commit.

I am personally also fine with the current GitHub button. Its styling may be left for some future PR / issue. If the others agree on this design, we can go with it.

**Comment by ViralBShah** [comment]
Merging. Can always revert if we need to.

File: _css/app.css [hunk 1]
```diff
@@ -438,26 +437,41 @@ img.ecosystem-image {
   margin-right: 20px;
 }
 
-.jumbotron {
-  background-image: url('../assets/infra/splash-img.png');
+.page-heading {
+  color: #fff;
+  padding: 3rem;
+  background-image: url('https://raw.githubusercontent.com/cormullion/graphics/ed1bbef52ae2081a5a192f23d35a0cb9b38b8d30/svg-banners/backsplash-3.svg');
```
Comment (cormullion) [inline _css/app.css:443]: Perhaps this should be copied across - the repo here is only temporary...

**Comment by abhishalya** [comment]
Also it seems on mobile the buttons are overlapping each other a bit.

File: _css/app.css (hunk 1 above)
Comment (tlienart) [reply _css/app.css:443]: I'm on it , PR incoming

**Comment by tlienart** [comment]
> Also it seems on mobile the buttons are overlapping each other a bit.

@abhishalya might be worth opening another issue with screenshots ?

**Comment by zdroid** [comment]
@tlienart I confirmed it, the fix is coming.

**Comment by zdroid** [comment]
@tlienart @ViralBShah It'd be interesting to check Google Analytics after this was implemented, e.g. for retention rate.

**Comment by fredrikekre** [comment]
The download button has a high saturation color when not hovered and a low saturation color when hovered. The documentation button is the other way around. Maybe high sat on hover for both? Also, the background is kinda dark, is it possible to lightening it up? Maybe remove the kinda thick black lines between the triangles would be one way to do that?

**Comment by zdroid** [comment]
@ViralBShah (or someone else with the access to the Analytics) Has there been any retention rate change for the front page since the past month? I'm just curious if the redesign had any measurable impact.

**Comment by ViralBShah** [comment]
How would one see that in analytics? The website traffic has all kinds of noise based on when articles show up on HN, releases are made, etc.

**Comment by zdroid** [comment]
@ViralBShah Behavior -> Site Content -> All Pages (also accessible via "Page Report"). `/` indicates the index there, as on this picture:
Screenshot from 2020-06-12 01-16-06
Pageviews are noisy, but the other metrics shouldn't be. Avg. time on page is obviously expected to increase, while bounce and exit rates are expected to decrease (although that may be a bit noisy).

**Comment by ViralBShah** [comment]
There's no noticeable change on any of these over the month before.

**Pull request merged**